dev-infrastructure/scripts/kv-permissions.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+PRINCIPAL_ID=$1
+RG_NAME=$2
+KV_NAME=$3
+
+KV_RESOURCE_ID=$(az keyvault show --name ${KV_NAME} --resource-group ${RG_NAME} --query id -o tsv 2>/dev/null)
+
+if [ -z "${KV_RESOURCE_ID}" ]; then
+    echo "Error: Key Vault resource ID for ${KV_NAME} in ${RG_NAME} could not be retrieved."
+    exit 0
+fi
+
+az role assignment create \
+    --role "Key Vault Secrets Officer" \
+    --assignee ${PRINCIPAL_ID} \
+    --scope ${KV_RESOURCE_ID} \
+    --only-show-errors
+
+az role assignment create \
+    --role "Key Vault Certificates Officer" \
```

Comment:
"Key Vault Certificates Officer: perform any action on the certificates of a key vault, excluding reading the secret and key portions, and managing permissions"

I wonder if we need "Key Vault Certificate User" which has this: Read entire certificate contents including secret and key portion. 

The part that confuses me is; are the certificate officer permission enough to be able to read certificates?

Reply:
certificate officer together with secret officer should be sufficient. but let me check

Reply:
you are right. i added the KV Certificage User as well

Reply:
Thank you.